pkg/launcher/launcher.go

```diff
@@ -129,12 +130,24 @@ func Launch(params Parameters) {
 		os.Exit(1)
 	}
 
+	// Setup scheme for Grafana
+	if err := grafana.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+
 	// Setup all Controllers
 	if err := controller.AddToManager(mgr); err != nil {
 		log.Error(err, "")
 		os.Exit(1)
 	}
 
+	// TODO: should I worry about the context here?
```

Comment:
No need to worry about. Just use `ctx.Background()` in `EnsureDashboardExported` and remove the method parameter.

Reply:
Okay.